### PR TITLE
chore: remove repetitive words

### PR DESCRIPTION
**Pull request by sellskin**
---

#### PR checklist

- [ ] Tests written/updated
- [ ] Changelog entry added in `.changelog` (we use [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
- [ ] Updated relevant documentation (`docs/` or `spec/`) and code comments
- [ ] Title follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) spec

### CODE REVIEW

File: docs/references/architecture/adr-102-rpc-companion.md
Comment (cason) [inline docs/references/architecture/adr-102-rpc-companion.md:1]: There is no much point on updating old ADRs content... but ok.

File: docs/references/rfc/tendermint-core/rfc-015-abci++-tx-mutation.md
Comment (cason) [inline docs/references/rfc/tendermint-core/rfc-015-abci++-tx-mutation.md:1]: The same applies to RFCs...

File: internal/consensus/reactor_test.go
Comment (cason) [inline internal/consensus/reactor_test.go:1]: I wouldn't change code comments. Please revert those changes.

File: spec/consensus/consensus-paper/IEEEtran.cls
Comment (cason) [inline spec/consensus/consensus-paper/IEEEtran.cls:1]: This is a Latex standard, downloaded from the internet...

**Review by cason — COMMENTED** [review]
Thank you for this, but please avoid changes in code.

File: internal/consensus/reactor_test.go
Comment (melekes) [reply internal/consensus/reactor_test.go:1]: why?

**Review by melekes — APPROVED** [review]
Thanks @sellskin ❤️

**Pull request merged**